CMakeLists.txt

```diff
@@ -139,3 +141,4 @@ if(CMAKE_BUILD_TYPE MATCHES Debug OR CMAKE_BUILD_TYPE MATCHES DEBUG)
 endif(CMAKE_BUILD_TYPE MATCHES Debug OR CMAKE_BUILD_TYPE MATCHES DEBUG)
 
 install(TARGETS hyprpaper)
+install(FILES ${CMAKE_BINARY_DIR}/systemd/hyprpaper.service DESTINATION "lib/systemd/user")
```

Comment:
> Should be `${CMAKE_INSTALL_LIBDIR}` instead of `lib`.

I tried, but https://github.com/hyprwm/hypridle/pull/46 happened, so I decided to go with full path instead.

Reply:
I just tried to build it again, and with "${CMAKE_INSTALL_LIBDIR}/systemd/user") 
the service ends up in /systemd/user/ instead of /usr/lib/systemd/user

Reply:
Ah yea you also need to add `include(GNUInstallDirs)` somewhere before the `install()`. Thought it was included already.

Reply:
Ok, I'll try rn

Reply:
Done. It works